### PR TITLE
Case summary

### PR DESCRIPTION
**Pull request by audreyli**
updated case and form summary pages so that they directly link to module and form view pages

### CODE REVIEW

**Comment by snopoke** [comment]
I think this needs to be rebased off master

File: corehq/apps/app_manager/templates/app_manager/ng_partials/case_summary_view.html [hunk 1]
```diff
@@ -120,7 +120,10 @@ <h1 style="font-size: 1.8em;">{% trans "Case Summary" %}</h1>
                             <td rowspan="{{ property.forms.length }}" ng-if="$first">{{ property.name }}</td>
                             <td>
                                 {% verbatim %}
-                                {{ getFormName(form.form_id, lang) }}
+                                    <a ng-href="/a/demo/apps/view/71d590784e9c5910947a5e7d1878f3b4/modules-{{ getModuleID(form.form_id, lang) }}/">
```
Comment (snopoke) [inline corehq/apps/app_manager/templates/app_manager/ng_partials/case_summary_view.html]: domain and app_id should be parameterized as well

**Review by snopoke — COMMENTED** [review]
This will be great

**Comment by proteusvacuum** [comment]
https://github.com/dimagi/commcare-hq/pull/14849

**Pull request closed without merging**